avocado-core/pom.xml

```diff
@@ -125,5 +125,21 @@
       <artifactId>scalatest_${scala.version.prefix}</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.seqdoop</groupId>
+      <artifactId>hadoop-bam</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.seqdoop</groupId>
+      <artifactId>cofoja</artifactId>
```

Comment:
Is this a separate dist of the cofoja library than what we were depending on elsewhere?  Did cofoja (do we know) ever get itself in working order for Java 1.8?  That was an issue, a while ago...